### DISCUSSION TITLE
评论的默认头像链接无法访问

### DISCUSSION THREAD

**Discussion by MinMinGuGu** [LIlGG/halo-theme-sakura#414] [Q&A]
![image](https://user-images.githubusercontent.com/46949010/193524439-aaf19d2c-e347-4077-8098-2d3ea2dfcf1d.png)
不输入QQ，链接`https://cn.gravatar.com/avatar?d=mm` request timed out。
暂时解决方案：在`sakura-comment.min.js`找到`//cn.gravatar.com/avatar`进行替换。

**Comment by LIlGG** [comment]
前往 Halo 后端 -> 系统 -> 博客设置 -> 评论设置，更改评论设置下的 `Gravatar 镜像源` 即可，无需手动替换

**Comment by MinMinGuGu** [comment]
主题设置 -> 资源 资源使用 CDN：开启
更改评论设置下的 Gravatar 镜像源，刷新文章页面，依然请求`https://cn.gravatar.com/avatar?d=mm`

**Comment by lukeewin** [comment]
> 前往 Halo 后端 -> 系统 -> 博客设置 -> 评论设置，更改评论设置下的 `Gravatar 镜像源` 即可，无需手动替换

请问更改为什么？
